src/app/services/project_service.js

```diff
@@ -2,6 +2,7 @@
 const angular = require('angular');
 const $ = require('jquery');
 const _ = require('lodash');
+const Fuse = require('fuse.js');
```

Comment:
i _think_ I ran into that `Object is not a constructor` error too. I can't _exactly_ remember how I resolved it, but my playing-around branch uses `import Fuse from 'fuse.js'`. Might be worth giving that a shot if you're still running into issues

Reply:
Yep this sorted it!